x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages.test.tsx

```diff
@@ -5,62 +5,96 @@
  * 2.0.
  */
 
-import { setMockValues } from '../../__mocks__/kea.mock';
+import { setMockValues, setMockActions } from '../../__mocks__/kea.mock';
 
 import React from 'react';
 
 import { shallow } from 'enzyme';
 
-import { EuiCallOut } from '@elastic/eui';
+import { EuiCallOut, EuiGlobalToastList } from '@elastic/eui';
 
-import { FlashMessages } from './flash_messages';
+import { FlashMessages, Callouts, Toasts } from './flash_messages';
 
 describe('FlashMessages', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
 
-  it('does not render if no messages exist', () => {
-    setMockValues({ messages: [] });
-
+  it('renders callout and toast flash messages', () => {
     const wrapper = shallow(<FlashMessages />);
-
-    expect(wrapper.isEmptyRender()).toBe(true);
+    expect(wrapper.find(Callouts)).toHaveLength(1);
+    expect(wrapper.find(Toasts)).toHaveLength(1);
   });
 
-  it('renders an array of flash messages & types', () => {
-    const mockMessages = [
-      { type: 'success', message: 'Hello world!!' },
-      {
-        type: 'error',
-        message: 'Whoa nelly!',
-        description: <div data-test-subj="error">Something went wrong</div>,
-      },
-      { type: 'info', message: 'Everything is fine, nothing is ruined' },
-      { type: 'warning', message: 'Uh oh' },
-      { type: 'info', message: 'Testing multiples of same type' },
-    ];
-    setMockValues({ messages: mockMessages });
+  describe('callouts', () => {
+    it('renders an array of flash messages & types', () => {
```

Comment:
Same as above - this is just an extra describe block, I recommend hiding whitespace changes